agent/command/git.go

```diff
@@ -707,13 +712,24 @@ func (c *gitFetchProject) fetchModuleSource(ctx context.Context,
 		repo:     repo,
 		branch:   "",
 		dir:      moduleBase,
+		method:   cloneMethod,
 	}
 	// Module's location takes precedence over the project-level clone
 	// method.
 	if strings.Contains(opts.location, "git@github.com:") {
 		opts.method = evergreen.CloneMethodLegacySSH
+	} else if opts.method == evergreen.CloneMethodAccessToken {
+		if appToken, ok := appTokens[opts.owner]; ok {
+			opts.token = appToken
+		} else {
+			appToken, err := comm.CreateInstallationToken(ctx, td, opts.owner, opts.repo)
+			if err != nil {
+				return errors.Wrapf(err, "creating installation token for '%s/%s'", opts.owner, opts.repo)
+			}
+			appTokens[opts.owner] = appToken
```

Comment:
Seems like we don't need to set this token because it's never used, unless it's for the module cache thing I mentioned in the above comment.